app/src/main/java/com/example/translatorapp/constant/constant.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.translatorapp.constant
+
+const val TAG_TRANSLATE = "translate"
+const val KEY_VISIBLE = "visible"
+const val KEY_TRANSLATE = "Translate"
+const val KEY_POSITION_TO = "positionTo"
+const val KEY_POSITION_FROM = "positionFrom"
+const val KEY_SETTING = "Setting"
+const val KEY_PITCH = "pitch"
+const val KEY_SPEED = "speed"
+const val KEY_DARK_MODE = "darkMode"
+const val KEY_RECREATE = "recreate"
```

Comment:
```suggestion
companion object Constants {
        const val TAG_TRANSLATE = "translate"
        const val KEY_VISIBLE = "visible"
        const val KEY_TRANSLATE = "Translate"
        const val KEY_POSITION_TO = "positionTo"
        const val KEY_POSITION_FROM = "positionFrom"
        const val KEY_SETTING = "Setting"
        const val KEY_PITCH = "pitch"
        const val KEY_SPEED = "speed"
        const val KEY_DARK_MODE = "darkMode"
        const val KEY_RECREATE = "recreate"
    }
```
Như này khi cần gọi sẽ clear hơn e nhé!
``Constants.xxx`` that vì ``xxx``